wadsrc/static/zscript/shared/dynlights.txt

```diff
@@ -28,7 +28,8 @@ class DynamicLight : Actor native
 		ColorFlickerLight, 
 		RandomColorFlickerLight
 	};
-	
+	native void SetOffset(Vector3 offset);
+
 	Default
```

Comment:
I'm sorry for being harsh, but are you kidding? There should be no removed lines, only two added, one with declaration and one empty.

Reply:
`There should be no removed lines, only two added, one with declaration and one empty.`It's already the case.